src/kv.c

```diff
@@ -0,0 +1,908 @@
+/*
+ * kv.c
+ *
+ * Copyright (C) 2021 wolfSSL Inc.
```

Comment:
2022 now :)

Reply:
good point!  will update in next PR, codebase-wide.

Reply:
To be fixed in subsequent PR.